src/components/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+  activeSlug: string | undefined;
+  search: string;
+}
+
+export const PersonLink: React.FC<Props> = ({
+  person: {
+    slug,
+    sex,
+    name,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  },
+  activeSlug,
+  search,
+}) => {
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': activeSlug === slug })}
+    >
+      <td>
+        <Link
+          to={{ pathname: `/people/${slug}`, search }}
+          className={classNames({ 'has-text-danger': sex === 'f' })}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother ? (
+          <Link
+            to={{ pathname: `/people/${mother.slug}`, search }}
+            className="has-text-danger"
+          >
+            {motherName}
+          </Link>
+        ) : (
+          motherName || '-'
+        )}
```

Comment:
You're repeating the same logic for `mother` and `father`. Consider creating a separate component for this part to avoid code duplication. This is a code quality suggestion to keep your code DRY (Don't Repeat Yourself).